install/bootstrap_livecd_remote_iso.md

```diff
@@ -122,7 +122,8 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
    ```
 
 1. <a name="set-up-site-link"></a>Set up the site-link, enabling SSH to work. You can reconnect with SSH after this step.
-   > **`NOTICE REGARDING DHCP`** If your site's network authority or network administrator has already provisioned an IPv4 address for your master node(s) external NIC(s), **then skip this step**.
+   > **`Note:`** If your site's network authority or network administrator has already provisioned a DHCP given IPv4 address for your master node(s) external NIC(s), **then skip this step**.
```

Comment:
I see all of the notes and warnings in this PR are both bolded and surrounded in backticks. In all of the /operations documentation (and in other product stream documentation), these are only bolded. 

I think we should keep all of these as bolded only to be consistent.

Reply:
Thats an interesting and valid observation. These install docs however had the Note and Warnings using this existing style already. At this time we're trying to limit from making too many style, format and content changes at once that might confuse readers. For now, we're going to move ahead with this style. We are adding a linter to the docs and will review this proposed change to remove the back tics and leave just the bold at a later time.